### PR TITLE
Created Hotel page UI

### PR DESCRIPTION
**Pull request by gbudjeakp**
Hotel page

Still needs some small tweaks.

### CODE REVIEW

File: client/src/pages/Hotels.js [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+	const classes = useStyles()
 	return (
-		<div>
-			<h1>This is the hotel page</h1>
-		</div>
+		<Grid className={classes.root}>
+		<Grid classes={{ container: classes.container }} container>
+		  <Grid className={classes.titleContainer} item xs={5}>
+			<Typography className={classes.header}>
+			  Find the best hotels<br></br> around the world.
```
Comment (johnsciutto) [inline client/src/pages/Hotels.js]: The `br`tag is self closing. So you don't need to add a `</br>`:

File: client/src/styles/FavoriteDestinations.js [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "flex-end",
+  },
+});
+
+export default useStyles
+||||||| 37e3814
+=======
```
Comment (johnsciutto) [inline client/src/styles/FavoriteDestinations.js]: Same as above.

File: client/src/styles/FavoriteDestinations.js [hunk 3]
```diff
@@ -1,3 +1,57 @@
+<<<<<<< HEAD
```
Comment (johnsciutto) [inline client/src/styles/FavoriteDestinations.js]: Make sure you delete the merge conflict markers.

File: client/src/styles/Profile.js [hunk 4]
```diff
@@ -1,3 +1,43 @@
+<<<<<<< HEAD
```
Comment (johnsciutto) [inline client/src/styles/Profile.js]: Make sure you delete the "merge conflict" markers from the file.

File: client/src/styles/Profile.js [hunk 5]
```diff
@@ -1,3 +1,43 @@
...
+   color: "#c5bec4",
+   marginRight: 30,
+   lineHeight: 2.2,
+   paddingTop: 2,
+  },
+  }));
+
+export default useStyles
+||||||| 37e3814
+=======
```
Comment (johnsciutto) [inline client/src/styles/Profile.js]: Same as above.

Comment (gbudjeakp) [reply client/src/styles/Profile.js]: I'll be pushing and effecting those changes.

**Review by sina-jamshidi — APPROVED** [review]
This looks good to me!

**Pull request merged**